php/basicphp/index.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
This file should be in a subdirectory name `public`

Reply:
Thanks. I placed the front controller file inside the public folder.